scenario-compiler/src/main/scala/pl/touk/nussknacker/engine/definition/component/parameter/defaults/SimpleLanguageDeterminer.scala

```diff
@@ -0,0 +1,36 @@
+package pl.touk.nussknacker.engine.definition.component.parameter.defaults
+
+import pl.touk.nussknacker.engine.api.definition._
+import pl.touk.nussknacker.engine.graph.expression.Expression
+
+trait SimpleLanguageDeterminer {
+
+  def calculateDefaultValue(
```

Comment:
It is a little bit supprising that simple language determiner calculate default value :) I'd rather expect that it would do some simple language determining ;-)

Reply:
fixed